Promact.Oauth.Server/src/Promact.Oauth.Server/appsettings.development.example.json

```diff
@@ -1,11 +1,15 @@
 ﻿{
-
-  "EmailCredential": {
+  "AppSettings": {
```

Comment:
I am not sure how you interpreted that I wanted everything in a single block. I have clearly stated that right now everything is in a single block and need a better way to organise. You have done the exact opposite of what I wanted.